client-v2/src/actions/__tests__/Cards.spec.ts

```diff
@@ -131,19 +116,10 @@ const move = (
   toType: string,
   toId: string,
   fromType: string,
-  fromId: string,
-  // sets the collection cap and allows a way to accept, reject, ignore the
-  // modal immediately
-  collectionCapInfo?: {
-    cap: number;
-    accept: boolean | null;
-  }
+  fromId: string
 ) => {
   const [uuid, id] = movedCardSpec;
-  const { dispatch, getState } = buildStore(
-    [uuid, id, undefined],
-    collectionCapInfo ? collectionCapInfo.cap : Infinity
-  );
+  const { dispatch, getState } = buildStore([uuid, id, undefined]);
```

Comment:
Is an integration test definitely preferable here? I'd favour unit tests for this behaviour if at all possible because the feedback loop is much faster when we're developing.

Reply:
Following on from our offline conversation, I've reinstated these tests but specified that they should be skipped for now, until we implement the relevant mock for the modal.